### PR TITLE
[feat] Prisma Integration

### LINKED ISSUE #3: Prisma Integration
We have decided to use prisma as our ORM. 

So let's start integrating prisma in our system. As a starter lets go with SQLite as no one have experience with docker and move to psql as we get more matured. Its better to integrate prisma as plugin and then use it as service everywhere. Ref provided

Ref:
https://www.prisma.io/fastify

Checklist

- [x] Prisma npm library added
- [x] Prisma config initialized
- [x] Gitignore sqllite file created
- [x] Integration with fastify
- [x] A dumy table record to verify integration

### PR DESCRIPTION
**Pull request by seramg**
Resolves #3

### CODE REVIEW

File: src/prisma.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const prismaPlugin: FastifyPluginAsync = fp(async (server, {}) => {
+  await prisma.$connect()
+  // Make Prisma Client available through the fastify server instance: server.prisma
+  server.decorate('prisma', prisma)
+
+  server.addHook('onClose', async (server) => {
+    await server.prisma.$disconnect()
+  })
+})
+async function main() {
```
Comment (akhilmhdh) [inline src/prisma.ts:22]: Remove this function and its call. Do a named export and avoid default export

**Review by akhilmhdh — CHANGES_REQUESTED** [review]
Where is the package.json file

File: src/prisma.ts (hunk 1 above)
Comment (seramg) [reply src/prisma.ts:22]: which function?
Did you mean the main function??
default export of prismaplugin must be avoided?

Comment (akhilmhdh) [reply src/prisma.ts:22]: 1. Yes
2. Use named export

**Comment by akhilmhdh** [comment]
@seramg Check ur commited files/ Package.json is missing from it

File: package.json [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
+      "start": "node ./dist/index.js",
+      "lint": "eslint ./src --ext .js,.ts",
+      "lint-fix": "eslint ./src --ext .js,.ts --fix",
+      "prepare": "husky install"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+      "@fastify/env": "^3.0.0",
```
Comment (akhilmhdh) [inline package.json:19]: Where is prisma in this dependencies?

File: src/prisma.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import fp from 'fastify-plugin'
+import { FastifyPluginAsync } from 'fastify'
+import { PrismaClient } from '@prisma/client'
+
+// Use TypeScript module augmentation to declare the type of server.prisma to be PrismaClient
+declare module 'fastify' {
```
Comment (akhilmhdh) [inline src/prisma.ts:6]: Move declaration to types.d.ts file

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

**Review by akhilmhdh — APPROVED** [review]

**Pull request merged**